items.md

```diff
@@ -137,6 +138,169 @@ Example: <https://dspace7.4science.it/dspace-spring-rest/#https://dspace7.4scien
+              "totalElements": 1
+            }
+          }
+        }
+      },
+      {
+        "id": "320c0492-de1d-4646-9e69-193d36b366e9",
+        "uuid": "320c0492-de1d-4646-9e69-193d36b366e9",
+        "name": "Biology",
+        "handle": "123456789/5285",
+        "metadata": [
+          {
+            "key": "dc.description.abstract",
+            "value": "",
+            "language": null
+          },
+          {
+            "key": "dc.title",
+            "value": "Biology",
+            "language": null
+          }
+        ],
+        "type": "collection",
+        "_links": {
+          "license": {
+            "href": "https://dspace7-internal.atmire.com/rest/api/core/collections/320c0492-de1d-4646-9e69-193d36b366e9/license"
+          },
+          "exportToZip": {
+            "href": "https://dspace7-internal.atmire.com/rest/api/core/collections/320c0492-de1d-4646-9e69-193d36b366e9/exportToZip"
+          },
+          "defaultAccessConditions": {
+            "href": "https://dspace7-internal.atmire.com/rest/api/core/collections/320c0492-de1d-4646-9e69-193d36b366e9/defaultAccessConditions"
+          },
+          "logo": {
+            "href": "https://dspace7-internal.atmire.com/rest/api/core/collections/320c0492-de1d-4646-9e69-193d36b366e9/logo"
+          },
+          "self": {
+            "href": "https://dspace7-internal.atmire.com/rest/api/core/collections/320c0492-de1d-4646-9e69-193d36b366e9"
+          }
+        },
+        "_embedded": {
+          "logo": null,
+          "defaultAccessConditions": {
+            "_embedded": {
+              "defaultAccessConditions": [
+                {
+                  "id": 28050,
+                  "name": null,
+                  "groupUUID": "9e1794b8-45e0-4869-9c2c-36c4b25ce856",
+                  "action": "DEFAULT_BITSTREAM_READ",
+                  "type": "resourcePolicy",
+                  "_links": {
+                    "self": {
+                      "href": "https://dspace7-internal.atmire.com/rest/api/authz/resourcePolicies/28050"
+                    }
+                  }
+                }
+              ]
+            },
+            "_links": {
+              "self": {
+                "href": "https://dspace7-internal.atmire.com/rest/api/core/collections/320c0492-de1d-4646-9e69-193d36b366e9/defaultAccessConditions"
+              }
+            },
+            "page": {
+              "number": 0,
+              "size": 1,
+              "totalPages": 1,
+              "totalElements": 1
+            }
+          }
+        }
+      }
+    ]
+  }
+```
+
+It returns all the mapped collections the item is included in
+
+On the item page, it should be referenced similar to:
+```json
+    "mappedCollections": {
+      "href": "https://dspace7-internal.atmire.com/rest/api/core/items/95e5d7d9-ef4e-4e35-86cc-07bfe2f0e355/mappedCollections"
+    }
+```
+
+**POST /api/core/items/<item:uuid>/mappedCollections?collection=<collection:uuid>**
```

Comment:
The param syntax in both POST and DELETE seems to not align with the implementation PR: https://github.com/DSpace/DSpace/pull/2282.  The implementation PR says this syntax is `POST /api/core/items/{itemUuid}/mappedCollections/{collectionUuid}`, which is the same syntax in @abollini's alternative #44.  

Should we sync up this PR with PR #44? Or just close this out in favor of #44?

For what it's worth, I prefer the syntax of PR #44 (and the current implementation), as `collection` doesn't need to be passed as a parameter here.